Fix header field value cannot include CR/LF

nmenag opened the pull request.

degzcs commented:
Sorry @nmenag  I was kind of busy these past months. If you want I can review this PR and merged it against master. Let me know

nmenag commented:
@degzcs brother I made the fork this repository and added new endpoints of Alegra API, updated some dependencies and last changed was an error when the username is very long.

degzcs commented:
@nmenag Cool! I just ran the specs and everything is working. I gonna update the gem.
